src/main/scala/rocket/CSR.scala

```diff
@@ -124,6 +124,17 @@ class MIP(implicit p: Parameters) extends CoreBundle()(p)
   val usip = Bool()
 }
 
+class Envcfg extends Bundle {
+  val stce = Bool()
+  val pbmte = Bool()
+  val zero54 = UInt(54.W)
```

Comment:
It is OK with me, but I have to say that senvcfg does not define these fields. Maybe a comment will clarify that?